README.md

```diff
@@ -48,78 +48,73 @@ add_action( 'hogan/include_modules', function( \Dekode\Hogan\Core $core ) {
+'hide_on_screen'                 => [],
+'fields_before_flexible_content' => [],
+'fields_after_flexible_content'  => [],
+```
 
-### Remove default field group
-If you dont want to use the default field group, or for some other reason want to setup a customized field group yourself, run this helper function in the theme setup to disable the default group.
+### Disable default field group
+If you don't want to use the default field group, or for some other reason want to setup a customized field group yourself, field groups can be disabled with a filter.
 
 ```php
-add_action( 'hogan/include_field_groups', function() {
-	hogan_deregister_default_field_group();
-} );
+add_filter( 'hogan/field_group/default/enabled', '__return_false' );
 ```
 
 ### Adding custom field groups
-Use the helper function `hogan_register_field_group()` in action `hogan/include_field_groups` to register custom field groups.
+Use the core function `register_field_group()` in action `hogan/include_field_groups` to register custom field groups.
+
 
 ```php
-function hogan_register_field_group( $name, $label, $modules = [], $location = [], $hide_on_screen = [], $fields_before_flexible_content = [], $fields_after_flexible_content = [] ) {
+add_action( 'hogan/include_field_groups', function( \Dekode\Hogan\Core $core ) {
+  $args = []; // Your field group args.
+  $core->register_field_group( $args );
+}, 10, 1 );
 ```
 
-#### Function arguments:
-- `$name`: Unique field group name
-- `$label`: Field group label
-- `$modules`: Array of modules you want to make available or null for all.
-- `$location`: The location parameter for [acf_add_local_field_group()](https://www.advancedcustomfields.com/resources/register-fields-via-php/)
-- `$hide_on_screen`: The hide_on_screen parameter for [acf_add_local_field_group()](https://www.advancedcustomfields.com/resources/register-fields-via-php/)
-- `$fields_before_flexible_content`: Custom ACF fields before the Flexible Content field.
-- `$fields_after_flexible_content`: Custom ACF fields after the Flexible Content field.
+See Customizing the default field group above for possible arguments.
 
 #### Example:
 
-This example demonstrates how to add a custom field group with just the text module for post type `page`.
+This example demonstrates how to add a custom field group with just the text module for post type `post`.
 ```php
-add_action( 'hogan/include_field_groups', function() {
-
-  $name = 'field_group_1';
-  $label = __( 'Field group Label', 'text-domain' );
-  $modules = [ 'text' ];
-  $location = [
-    [
+add_action( 'hogan/include_field_groups', function( \Dekode\Hogan\Core $core ) {
+  $core->register_field_group( [
+    'name' => 'field_group_1',
+    'title' => __( 'Field group title', 'text-domain' ),
+    'modules' => [ 'text' ],
+    'location' => [
       [
-        'param' => 'post_type',
-        'operator' => '==',
-        'value' => 'page',
+        [
+          'param' => 'post_type',
```

Comment:
Missing `'operator' => '==',`

Reply:
`==` is the default acf operator and is not needed here.
<img width="286" alt="screen shot 2018-04-24 at 13 27 26" src="https://user-images.githubusercontent.com/16900754/39184314-3f10345a-47c3-11e8-9cc8-c8aa3bb4475c.png">

Reply:
Fixed.